src/partials/sample.hbs

```diff
@@ -0,0 +1,21 @@
+{{#each pets}}
+  <div class="pet">
+    <div class="photo-column">
+      <img src="{{photo}}">
+    </div>
+
+    <div class="info-column">
+      <h2>{{name}} <span class="species">({{species}})</span></h2>
+
+      {{#if favFoods}}
+      <h4 class="headline-bar">Favorite Foods</h4>
```

Comment:
You cant add H4 after H2, follow sequence of H1, H2, H3..

Reply:
this file is not being used and removed